package.json

```diff
@@ -6,14 +6,16 @@
     "ios": "expo start --ios",
     "lint": "./node_modules/.bin/eslint __tests__ src --ext .js,.jsx,.ts,.tsx",
     "start": "expo start",
-    "test": "jest",
+    "test": "node_modules/.bin/jest",
```

Comment:
how does this differentiate from `"test": "jest"`?

Reply:
do you know the difference regarding the two paths?

Reply:
"jest" is installed globally on my machine ? and "node_modules/.bin/jest" is locally just in my project node modules?

Reply:
should we define `./node_modules..` like in the `"lint"` command, to ensure that both work the same regarding their path of execution?

Reply:
so adding the dot but also this part `__tests__ src --ext .js,.jsx,.ts,.tsx"` ?

Reply:
no these options are part of eslint cli and not jest. you can read about them here https://eslint.org/docs/user-guide/command-line-interface